boards/microbit/Makefile.include

```diff
@@ -3,6 +3,7 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
 
 # for this board we support flashing via openocd or pyocd
 PROGRAMMER ?= openocd
+PROGRAMMERS_SUPPORTED ?= openocd jlink pyocd
```

Comment:
`openocd` and `jlinK` are added in `nclude $(RIOTBOARD)/common/nrf51/Makefile.include`